modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/persistence/RocksDbKeyValueStorage.java

```diff
@@ -427,7 +436,7 @@ private void updateKeysIndex(WriteBatch batch, byte[] key, long curRev) {
      * @throws RocksDBException If failed.
      */
     private void fillAndWriteBatch(WriteBatch batch, long newRev, long newCntr, @Nullable HybridTimestamp ts) throws RocksDBException {
-        try (WriteOptions opts = new WriteOptions()) {
+        try (WriteOptions opts = new WriteOptions().setDisableWAL(true)) {
```

Comment:
Could you please leave a comment on why we need (and can) disable the WAL to make it explicit?